getting-started.md

```diff
@@ -0,0 +1,51 @@
+# Getting Started With Accessibility
+
+- [Getting Started With Accessibility](#getting-started-with-accessibility)
+  - [What do we mean by ‘accessibility’?](#what-do-we-mean-by-accessibility)
+  - [What access do we need to provide?](#what-access-do-we-need-to-provide)
+    - [Screen readers](#screen-readers)
+    - [Magnification](#magnification)
+    - [Keyboard accessibility](#keyboard-accessibility)
+    - [Use of colour](#use-of-colour)
+  - [How do we do it?](#how-do-we-do-it)
+  - [How can I find out more?](#how-can-i-find-out-more)
+
+This guide is intended to give you a basic understanding of what accessibility is, why it matters, and the fundamental things you need to know in order to build web pages and apps that are accessible to everyone.
+
+## What do we mean by ‘accessibility’?
+
+Everybody has access needs. If you want to go to a shop, it needs to have a front door; if you want to look at a website, it needs to load on your phone or computer. If a shop’s front door is locked, or a website doesn’t load on your device, your ability to access the things you want has been limited.
+
+Accessibility is all about making sure that the things we build and the services we provide can meet the access needs of the widest possible range of people. Guardian journalism should be open to everyone, our internal tools should be accessible to all our colleagues, and we want to make sure that we don’t accidentally create barriers to people with differing needs.
```

Comment:
I wonder if we should call out and say `especially people with disabilities`. It's easy to miss this and then have people ask whether site performance or browser support is an accessibility concern... and if so, why aren't we talking about bundle sizes or polyfills in this guide?

Reply:
I'll add, though will make it 'disabled people' as that's the preferred language in the UK disability community.